packages/edit-site/src/components/page-templates/dataviews-templates.js

```diff
@@ -237,13 +187,75 @@ export default function DataviewsTemplates() {
 			{
 				header: __( 'Author' ),
 				id: 'author',
-				render: ( { item } ) => <AuthorField item={ item } />,
+				getValue: ( { item } ) => item.author_text,
+				render: ( { item } ) => {
+					return <AuthorField item={ item } />;
+				},
 				enableHiding: false,
-				enableSorting: false,
+				enableSorting: true,
```

Comment:
We can remove this, as the sorting is enabled by default.

Reply:
Removed.